package.json

```diff
@@ -13,6 +13,7 @@
   },
   "prettier": "@valora/prettier-config",
   "devDependencies": {
+    "@actions/github": "^5.0.0",
```

Comment:
is this used in the CI action or something?  I wasn't sure where we were actually using it.

Reply:
I was, I tried using it but ended up going a different route, removing